analytics_data_api/v0/tests/views/test_learners.py

```diff
@@ -185,3 +189,227 @@ def test_bad_course_id(self):
+        self.assert_learners_returned(response, expected_learners)
+
+    @ddt.data(
+        ([{'username': 'a'}, {'username': 'b'}], None, None, [{'username': 'a'}, {'username': 'b'}]),
+        ([{'username': 'a'}, {'username': 'b'}], None, 'desc', [{'username': 'b'}, {'username': 'a'}]),
+        ([{'username': 'a'}, {'username': 'b'}], 'username', 'desc', [{'username': 'b'}, {'username': 'a'}]),
+        ([{'username': 'a'}, {'username': 'b'}], 'email', 'asc', [{'username': 'a'}, {'username': 'b'}]),
+        ([{'username': 'a'}, {'username': 'b'}], 'email', 'desc', [{'username': 'b'}, {'username': 'a'}]),
+        (
+            [{'username': 'a', 'discussions_contributed': 0}, {'username': 'b', 'discussions_contributed': 1}],
+            'discussions_contributed', 'asc', [{'username': 'a'}, {'username': 'b'}]
+        ),
+        (
+            [{'username': 'a', 'discussions_contributed': 0}, {'username': 'b', 'discussions_contributed': 1}],
+            'discussions_contributed', 'desc', [{'username': 'b'}, {'username': 'a'}]
+        ),
+        (
+            [{'username': 'a', 'problems_attempted': 0}, {'username': 'b', 'problems_attempted': 1}],
+            'problems_attempted', 'asc', [{'username': 'a'}, {'username': 'b'}]
+        ),
+        (
+            [{'username': 'a', 'problems_attempted': 0}, {'username': 'b', 'problems_attempted': 1}],
+            'problems_attempted', 'desc', [{'username': 'b'}, {'username': 'a'}]
+        ),
+        (
+            [{'username': 'a', 'problems_completed': 0}, {'username': 'b', 'problems_completed': 1}],
+            'problems_completed', 'asc', [{'username': 'a'}, {'username': 'b'}]
+        ),
+        (
+            [{'username': 'a', 'problems_completed': 0}, {'username': 'b', 'problems_completed': 1}],
+            'problems_completed', 'desc', [{'username': 'b'}, {'username': 'a'}]
+        ),
+        (
+            [{'username': 'a', 'videos_viewed': 0}, {'username': 'b', 'videos_viewed': 1}],
+            'videos_viewed', 'asc', [{'username': 'a'}, {'username': 'b'}]
+        ),
+        (
+            [{'username': 'a', 'videos_viewed': 0}, {'username': 'b', 'videos_viewed': 1}],
+            'videos_viewed', 'desc', [{'username': 'b'}, {'username': 'a'}]
+        ),
+    )
+    @ddt.unpack
+    def test_sort(self, learners, order_by, sort_order, expected_users):
+        for learner in learners:
+            learner['course_id'] = self.course_id
+        self.create_learners(learners)
+        params = dict()
+        if order_by:
+            params['order_by'] = order_by
+        if sort_order:
+            params['sort_order'] = sort_order
+        response = self._get(self.course_id, **params)
+        self.assert_learners_returned(response, expected_users)
+
+    def test_pagination(self):
+        usernames = ['a', 'b', 'c', 'd', 'e']
+        page_url = 'http://testserver/api/v0/learners/?{course_query}&page={page}&page_size={page_size}'
```

Comment:
I was a bit confused by this variable name initially.  Can it be renamed to something like `expected_page_url_template` instead?